### PR TITLE
Expose CG queries and add changelog.

### PR DESCRIPTION
**Pull request by mikelehen**

### CODE REVIEW

File: firebase-firestore/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+  given collection ID using the `FirebaseFirestore.collectionGroup()` method.
+
+# 18.2.0
 - [feature] Added `FieldValue.increment()`, which can be used in `update()`
   and `set(..., SetOptions.merge())` to increment or decrement numeric field
   values safely without transactions.
+- [feature] Added functional interface `FirebaseFirestore.runBatch()`, similar
+  to `FirebaseFirestore.runTransaction()`, which allows a developer to focus
+  on the mutations of the batch rather than on creating and committing the
+  batch.
```
Comment (mikelehen) [inline firebase-firestore/CHANGELOG.md]: This ended up in the public changelog but not here, so I added it.  I believe this was part of @vkryachko's work to improve our Kotlin API.

Comment (rsgowman) [reply firebase-firestore/CHANGELOG.md]: Ah, sorry about this. Yeah, this was part of the public changelog, which I was *supposed* to backport here earlier, but didn't end up doing until today. (I noticed while reviewing one of Gil's PRs.)

Comment (rsgowman) [reply firebase-firestore/CHANGELOG.md]: I guess you can remove this, since we've got it in another PR now.

**Comment by mikelehen** [comment]
Thanks.  I'll re-merge with master and fix up the changelog once https://github.com/firebase/firebase-android-sdk/pull/342 lands

**Pull request merged**